test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -128,4 +128,49 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
       AccountRootFlag.checkFlag(AccountRootFlag.LSF_REQUIRE_AUTH, flags!),
     )
   })
+
+  it('requireAuthorizedTrustlines/allowUnauthorizedTrustlines - rippled', async function (): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+    // GIVEN an existing testnet account
+    await issuedCurrencyClient.requireAuthorizedTrustlines(wallet)
```

Comment:
I think perhaps the solution to avoid creating too many new faucet wallets (which really slows down the integration tests)  is to change the final check in this test to `assert.isFalse(AccountRootFlag.checkFlag(AccountRootFlag.LSF_REQUIRE_AUTH, flags!))`  .  `NO_FLAGS` is a clever approach, but I think should make the check with the flag-specific bit mask to avoid checking anything other than the flag in question.